nx-X11/programs/Xserver/hw/nxagent/Screen.c

```diff
@@ -336,6 +336,11 @@ Bool nxagentIsParentOf(Display *d, Window possible_parent, Window candidate)
   }
 }
 
+/*
+ * In contrast to its name this function is here to take care of the
+ * iconfied window. Some window managers tend to do 'interesting'
```

Comment:
s/iconfied/iconified/ (which window do you refer to here? The nxagent window? Please explain better)